projects/loopback.json

```diff
@@ -78,68 +162,27 @@
     "#sprint85": null,
     "#sprint86": null,
     "#sprint87": null,
-    "support": "eeeeee",
-    "#tbr": "eeeeee",
+    "documentation": null,
+    "Needs Doc": null, 
```

Comment:
might not be possible at this time though because I guess @crandmck Would have some items already using the uppercase label